controller_manager/test/test_load_controller.cpp

```diff
@@ -49,198 +66,130 @@ TEST_F(TestLoadController, load_controller_failed_init)
     nullptr);
 }
 
-TEST_F(TestLoadController, load_and_configure_one_known_controller)
```

Comment:
Did you remove this test or moved it somewhere?

Reply:
Line 107

I just restructured things
1. failed controller loading tests
2. 1 loaded controller tests
3. more loaded controller tests